.changelog/c898d340-9c15-4b9a-a5da-93a340a9d5f8.json

```
@@ -0,0 +1,9 @@
+{
+  "id": "c898d340-9c15-4b9a-a5da-93a340a9d5f8",
+  "type": "bugfix",
+  "collapse": true,
```

Nothing to collapse, the change affects one module

```suggestion
  "collapse": false,
```